docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md

```diff
@@ -16,9 +16,9 @@ topic_type:
 ---
 # ICorDebugProcess11::EnumerateLoaderHeapMemoryRegions Method
 
-[Supported in .NET Framework 4.5.2 and later versions.]  
+[Supported in .NET Core 5 and later versions.]  
   
- Configures how the debugger handles in-memory updates to metadata within the target process.  
+ Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap-loader" command. 
```

Comment:
Previously I remember seeing that comment about handling in-memory updates in three different spots in the documentation you added. Did we get all the places updated?

Reply:
I looked in the files that I have changed and it looks like all of them say the correct info.

Reply:
I believe so, I searched through the files I have changed